Sources/App/githubRoutes.swift

```diff
@@ -0,0 +1,59 @@
+import Vapor
+
+public func githubRoutes(router: GithubCommandRouter) throws {
+    router.register(command: "@vapor-bot", "test performance") { (req) -> Future<HTTPStatus> in
+        return try req.content.decode(GithubWebhook.self).flatMap { webhook in
+            guard let comment = webhook.comment, comment.user.login != "vapor-bot" else {
+                return req.future(.ok)
+            }
+            let repo = webhook.repository
+            let issue = webhook.issue
+            
+            let circle = try req.make(CircleCIService.self)
+            let github = try req.make(GithubService.self)
+            
+            guard let pullRequest = webhook.issue.pullRequest else {
+                return github.postComment(
+                    repo: repo.fullName,
+                    issue: issue.number,
+                    body: "Unknown command",
+                    on: req
+                ).transform(to: .ok)
+            }
+            
+            return try req.client().get(pullRequest.url).flatMap { response -> Future<String> in
+                let pullRequestHead = try response.content.syncDecode(GithubPullRequest.self).head
+                let branchName = pullRequestHead.name
+                
+                return circle.start(job: "linux-performance", repo: repo.fullName, branch: branchName, on: req)
+            }.flatMap { _ -> Future<CreateCommentResponse> in
+                return github.postComment(
+                    repo: repo.fullName,
+                    issue: issue.number,
+                    body: "Starting performance test",
+                    on: req
+                )
+            }.transform(to: .ok)
+        }
+    }
+    
+    router.register(command: "@vapor-bot") { req -> Future<HTTPStatus> in
```

Comment:
I don't think you want this here, but you'd want this in a "catchall" scenario.

Maybe have a flow like this:
```
Decode webhook
Check for prefix (@vapor-bot)
Bail out if no prefix
"Route" command
Based on a found command, return the command, or the catchall
```